### DISCUSSION TITLE
How to generate 10 similar images from one random prompt?

### DISCUSSION THREAD

**Discussion by Grim408** [adieyal/sd-dynamic-prompts#724] [Q&A]
Hi,

I want A1111 to make 10 similar images from one random prompt (with wildcards) but using different seeds for each image. 

In Comfy UI I just set the '**Batch size**' to 10 in '**Empty latent image**' and Comfy generates 10 images from the same prompt but with different seeds and then continues with second random prompt. 

'**Batch size**' in A1111 does not do the same thing, it generates 10 different prompts.

Is there a way to make this possible in A1111?

Thank you.

**Comment by uorufu** [comment]
In automatic1111 the way it handles prompts it is a bit tricky.

If you dont use things like adetailer it is pretty easy, but with extensions like adetailer that include extra prompting, I think you can only achieve this with a global variable like I did in my wildcards fork (which has a setting to 'lock' down the random wildcard generation for all seeds by using a global variable).

Any other way and adetailer will generate different results.

**Comment by akx** [comment]
Does https://github.com/adieyal/sd-dynamic-prompts?tab=readme-ov-file#fixed-seed help you?
